app/models/spree_avatax_certified/address.rb

```diff
@@ -137,5 +137,12 @@ def resolve_stock_location
       package = Spree::Stock::Coordinator.new(@order).packages.find { |package| package.line_items.any? {|li| li.id == @line.id} }
       package.stock_location
     end
+
+    def resolve_item_ship_to_address
+      shipment = @order.shipments.select { |s| s.line_items.include?(@line) }
```

Comment:
It would be good to have a strategy to avoid iterating over all (worst case) the shipments and all the line items in each shipment (again, worst case) for every line item in the order. For big orders this could be a problem.

What about a dictionary? I'm sure there are other solutions too